crates/sparrow-instructions/src/evaluators/aggregation/boolean/last_boolean_evaluator.rs

```diff
@@ -161,25 +160,24 @@ impl LastBooleanEvaluator {
         // want to store.
         Self::ensure_entity_capacity(token, key_capacity);
 
-        let result: BooleanArray =
-            if let Some(input_valid_bits) = BitBufferIterator::array_valid_bits(input) {
-                izip!(key_indices.values(), input_valid_bits, 0..)
-                    .map(|(entity_index, input_is_valid, input_index)| {
-                        Self::update_accum(
-                            token,
-                            *entity_index,
-                            input_is_valid,
-                            input.value(input_index),
-                        )
-                    })
-                    .collect::<anyhow::Result<BooleanArray>>()?
-            } else {
-                izip!(key_indices.values(), 0..)
-                    .map(|(entity_index, input_index)| {
-                        Self::update_accum(token, *entity_index, true, input.value(input_index))
-                    })
-                    .collect::<anyhow::Result<BooleanArray>>()?
-            };
+        let result: BooleanArray = if let Some(input_valid_bits) = input.nulls() {
```

Comment:
https://docs.rs/arrow/latest/arrow/array/trait.Array.html#tymethod.nulls

Wouldn't this be the...non-valid bits then? I still don't know.